spec/active_record/connection_adapters/oracle_enhanced_structure_dump_spec.rb

```diff
@@ -152,6 +152,19 @@ class ::TestPost < ActiveRecord::Base
       expect(dump).to match(/CREATE TABLE \"BARS\" \(\n \"ID\" NUMBER\(38,0\) NOT NULL,\n \"SUPER\" RAW\(255\) GENERATED ALWAYS AS \(HEXTORAW\(TO_CHAR\(ID\)\)\) VIRTUAL/)
     end
 
+    it "should dump NCLOB columns" do
+      skip "Not supported in this database version" unless @oracle11g_or_higher
```

Comment:
NCLOB data type exists at least Oracle 9i 9.0.1. Please remove this `skip` statement.